compact/tracking/definitions.xml

```diff
@@ -174,9 +174,13 @@
   <plugins>
     <plugin name="epic_FileLoader">
       <arg value="cache:$DETECTOR_PATH:/opt/detector"/>
-      <arg value="file:calibrations/materials-map.cbor"/>
+      <arg value="file:calibrations/material-map-brycecanyon.cbor"/>
       <arg value="url:https://eicweb.phy.anl.gov/EIC/detectors/athena/uploads/77bd16bef5d811f31256279a3601c9d5/material-maps.cbor"/>
     </plugin>
   </plugins>
+  <define>
+    <constant name="materials-map" value="calibrations/material-map-brycecanyon.cbor" type="string"/><!-- EICrecon older than 1.6.0 -->
```

Comment:
```suggestion
```

Reply:
We can strip the entire `define` environment too then.

Reply:
We'd still need `material-map` constant.